pkg/client/client.go

```diff
@@ -992,7 +992,12 @@ func (c *Client) WaitForDaemonSetRollout(ctx context.Context, ds *appsv1.DaemonS
 		for _, node := range nodeList.Items {
 			for _, condition := range node.Status.Conditions {
 				if condition.Type == v1.NodeReady && condition.Status == v1.ConditionTrue {
-					nodeReadyCount++
+					labelMap := node.Labels
+					for name, value := range labelMap {
```

Comment:
You can make use of LabelSelector in `metav1.ListOptions`. It is as equvalent as `kubectl get nodes -l "kubernetes.io/os=linux"`

Reply:
Done

Reply:
I am kindly think `kubernetes.io/os=linux` maybe not proper, especially  for windows nodes, we should be able to schedule node-exporter to each node, including windows node which don't have `kubernetes.io/os=linux`  label

Reply:
@juzhao node_exporter is meant only for Linux.